### ISSUE TITLE
admin.hub.nuxt.com has a timeout occurred when cache is large

### ISSUE THREAD

**Issue by oritwoen** [nuxt-hub/core#398]
**Describe the bug**
In the hub panel, you can't access the cache and a timeout occurs. This only happens with large databases for CACHE. When I clear the entire KV from the cache or have little of it, everything works correctly.

![Zrzut ekranu z 2024-12-16 23-46-39](https://github.com/user-attachments/assets/e97211e1-f12c-448d-9bb7-bf64643f1b48)
![Zrzut ekranu z 2024-12-16 23-46-05](https://github.com/user-attachments/assets/a8a02861-438f-496e-87b5-93a8e498b1ec)

**Comment by atinux** [comment]
180k records, I see, very interesting amount, will have a look to improve!

**Comment by oritwoen** [comment]
Yes, I connected Nuxt Hub to a client's website that is several years old, has tens of thousands of entries and relations. For now, it is based on WordPress's `wp-json` with a cache for each tag, category and entry.

The traffic there is at the level of 40-100k visits per day. And NuxtHub works great, but the only such a nuisance is that timeouts are in the panel for the cache.

Although recently I am leaning towards connecting `db0` from unjs and connecting to the database directly to connect to mysql directly, bypassing `wp-json` and not doing so much caching because with such traffic it gets very expensive.

It would be the best if `hyperdrive` supported `mysql` (I'm waiting xD) and then everything would be simpler.

I wrote a bit, but this is the context. If you want more information, logs or statistics from such high traffic, let me know, I will send it.

**Comment by luminous8** [comment]
I'm having the same issue while having way lower values (except storage)
<img width="1395" alt="Capture d’écran 2025-01-09 à 11 06 57" src="https://github.com/user-attachments/assets/5f784708-0f0d-4336-b9a0-ca9085223def" />

**Comment by luminous8** [comment]
Hey @atinux

Have you had a chance to look at it?

**Comment by atinux** [comment]
Still on my roadmap, sorry for the delay

**Comment by luminous8** [comment]
I really need to clear the clear the cache to fix some SEO issues

I have lots of KV entries
<img width="527" alt="Image" src="https://github.com/user-attachments/assets/a177bd58-5463-4f7a-bcfc-9d56c29f13ec" />

i thought of deleting the KV store in cloudflare directly but i cannot find a way to bind a KV store in NuxtHub

any workaround idea that could do the job easily here?

**Comment by atinux** [comment]
Quick and dirty way would be to go to your Cloudflare project:

1. unbind the KV namespace
2. Go to your KV namespace and delete it
3. Create again the namespace with the same name
4. Go back to your CF project and bind it again
5. Retry latest deployment

**Comment by RihanArfan** [comment]
With the introduction of NuxtHub v0.10 which supports multiple clouds and deployments directly from the cloud provider's tooling, we're sunsetting NuxtHub Admin now. https://hub.nuxt.com/changelog/nuxthub-multi-vendor

You can manage cache entries directly from the cache Cloudflare KV resource in the Cloudflare dashboard.
